MAUI/DataGrid/Row-height-Customization.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: post
+title: Row Height Customization in .NET MAUI DataGrid control | Syncfusion
+description: Learn here all about Row Height Customization support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Row Height Customization in .NET MAUI DataGrid (SfDataGrid)
+
+The data grid provides an option to customize the header row height and the row height of all the grid rows or a particular row. To achieve this customization, see the following sections:
+
+## Customize HeaderRowHeight 
+
+The data grid allows you to customize the height of the header row by setting the [SfDataGrid.HeaderRowHeight](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_HeaderRowHeight) property. The default value of this property is 55. This property responds to runtime changes, so it can be customized. Setting `SfDatagrid.HeaderRowHeight` to zero will collapse the header row in the view.
+
+To customize the header row height, follow the code example:
+
+{% highlight c# %}
+//Customizing header row height in SfDataGrid
+dataGrid.HeaderRowHeight = 50; 
+{% endhighlight %}
+
+## Customize RowHeight for all rows
+
+The data grid allows you to customize the height of grid rows in the scrolling region by setting the [SfDataGrid.RowHeight](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_RowHeight) property. The default value of this property is 47. This property responds to runtime changes, so it can be customized. Setting this property will change the height of all the rows in a body region with a common value. Setting `SfDataGrid.RowHeight` to zero will collapse all rows in the grid.
```

Comment:
The data grid allows you to customize the height of data rows in the scrolling region by setting the

Reply:
Remove the below line,
Setting this property will change the height of all the rows in a body region with a common value.

Reply:
Changed the above line and removed the below mentioned line.